src/main/scala/cromwell/binding/package.scala

```diff
@@ -13,5 +13,7 @@ import cromwell.binding.values.WdlValue
  */
 
 package object binding {
+  type Wdl = String
```

Comment:
Not a big deal, but my objection was to just plain `Wdl` as I thought that was too generic (it's the prefix of an awful lot of our recently created classes).  I'd be fine with more specific names like the two I suggested (though not necessarily only those two) or just leaving this un`type`d since I can't imagine too many spots in the code where it would be appropriately used anyway.